src/main/java/com/hms/flexyosisoftconnector/dataserver/OSIsoftServer.java

```diff
@@ -23,18 +23,27 @@
  */
 public class OSIsoftServer {
 
+  /** Boolean to represent when the last sent message was succesfully sent. */
   private static boolean connected = true;
 
   public static final int LINK_ERROR = 32601;
+
+  /** Ewon error code representing a send error */
   public static final int SEND_ERROR = 32603;
-  public static final int JSON_ERROR = -100;
 
+  /** Ewon error code representing a JSON error */
+  public static final int JSON_ERROR = -100;

```

Comment:
You're adding a new constant in the 'Added missing javadocs to OSIsoftServer' commit

Reply:
that got dropped down from line 30 to 35

Reply:
Oh, I see the Git diff is just weird looking